development-vm/replication/aws.sh

```diff
@@ -10,9 +10,9 @@ aws_auth() {
 
     # setup AWS access
     SESSION_NAME=$(whoami)-$(date +%d-%m-%y_%H-%M)
-    ROLE_ARN=$(awk '/profile govuk-integration/ {profile=1} /role_arn/ && profile==1 {print $3; exit}' ~/.aws/config)
-    MFA_SERIAL=$(awk '/profile govuk-integration/ {profile=1} /mfa_serial/ && profile==1 {print $3; exit}' ~/.aws/config)
-    SOURCE_PROFILE=$(awk '/profile govuk-integration/ {profile=1} /source_profile/ && profile==1 {print $3; exit}' ~/.aws/config)
+    ROLE_ARN=$(awk '/profile govuk-infrastructure-integration/ {profile=1} /role_arn/ && profile==1 {print $3; exit}' ~/.aws/config)
+    MFA_SERIAL=$(awk '/profile govuk-infrastructure-integration/ {profile=1} /mfa_serial/ && profile==1 {print $3; exit}' ~/.aws/config)
+    SOURCE_PROFILE=$(awk '/profile govuk-infrastructure-integration/ {profile=1} /source_profile/ && profile==1 {print $3; exit}' ~/.aws/config)
```

Comment:
The format the docs recommend for the AWS profile is `govuk-<environment>`, which is compatible with what's here. While you can use anything in your `~/.aws/config` file, this change will break the script for those who use `govuk-integration` as the name of the profile.

Reply:
https://docs.publishing.service.gov.uk/manual/aws-cli-access.html#1-create-awsconfig

Reply:
(maybe recommend is too strong, it's just the example given)

Reply:
This is true, fair point. If anyone did have `govuk-integration` in their profile, do you think that'd cause them problems elsewhere where that profile might be used, and could provide an incentive to update the name to be consistent with the account? If not, I'll close this PR.

Reply:
I think the profile names are entirely arbitrary, it's just that we do use them in a few disparate places.

Another place where the `govuk-<environment>` format is assumed is the govukcli tool in govuk-aws: https://github.com/alphagov/govuk-aws/blob/master/tools/govukcli#L316

Reply:
Bump, any update on this?